packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -1,13 +1,39 @@
 import classNames from 'classnames';
 import React from 'react';
-import PropTypes from 'prop-types';
 import { matchStringByChar, getPosition } from '@deriv/shared';
 import Icon from '../icon';
 import Input from '../input';
-import DropdownList from '../dropdown-list';
+import DropdownList, { TItem } from '../dropdown-list';
 import { useBlockScroll } from '../../hooks/use-blockscroll';
 import { getEnglishCharacters } from '../../../utils/helper';
 
+type TAutocompleteProps = {
+    autoComplete: string;
+    className: string;
+    disabled?: boolean;
+    dropdown_offset: string;
+    error: string;
+    has_updating_list: boolean;
+    hide_list: boolean;
```

Comment:
we can make these optional since we are setting default value for them.
same for `is_list_visible`, `show_list`